js/desktopCapturer/index.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+const electron = require('electron');
+const BrowserWindow = electron.BrowserWindow;
+const ipc = electron.ipcMain;
+const path = require('path');
+const fs = require('fs');
+const log = require('../log.js');
+const logLevels = require('../enums/logLevels.js');
+const { isMac, isWindowsOS } = require('./../utils/misc.js');
+
+let screenPickerWindow;
+let preloadWindow;
+let eventId;
+
+let windowConfig = {
+    width: 580,
```

Comment:
can we get this dynamically?

Reply:
Dynamically with reference to display resolution or HTML content?

Reply:
For creating a dynamic window with respect to HTML content. first, we need to open the window with some value for `width & height` and then we need to publish an event from `render process` to the `main process` when the DOM is loaded to resize the window.

Currently, the `width and height` of the window is exactly the same as the HTML content. I don't think that we need to dynamically update the window `width or height`